packages/oscal-react-library/src/components/OSCALControl.tsx

```diff
@@ -241,9 +242,44 @@ const OSCALControl: React.FC<OSCALControlProps> = (props) => {
     );
   }
 
+  const position = props?.modificationAlters
+    ?.find((item) => item["control-id"] === props.control.id)
+    ?.adds?.flatMap((item) => item["position"])
+    .filter((part) => part)[0];
+
   const label = propWithName(control.props, "label")?.value;
   const controlOrParentWithdrawn = withdrawn || isWithdrawn(control);
 
+  let modificationDisplayBefore = modificationDisplay;
+  let modificationDisplayAfter = modificationDisplay;
+  let modificationDisplayEnding = modificationDisplay;
+  let modificationDisplayStarting = modificationDisplay;
```

Comment:
> but you can see that when I defined modificationDisplayBefore, I have to turn off all the other modificationDisplay

```suggestion
  const modificationDisplayBefore = position === Position.BEFORE ? modificationDisplay : null;
  const modificationDisplayAfter = position === Position.AFTER ? modificationDisplay : null;;
  const modificationDisplayEnding = position === Position.ENDING ? modificationDisplay : null;;
  const modificationDisplayStarting = position === Position.STARTING ? modificationDisplay : null;;
```

I am pretty sure this is equivelent to what you are doing right? If `position == BEFORE`, all other the other variables will be `null`. You also could just do this check everywhere you put these variables, but that is up to personal preference. 

This makes it so we do not have to use `let` and it makes the code a lot clearer. I really think there must be a cleaner way to deal with this other than the `switch`.